src/main/java/de/adesso/anki/battle/protocol/commands/PutMineCommand.java

```diff
@@ -1,27 +1,23 @@
-package com.commands;
-
-import de.adesso.anki.battle.world.World;
-import de.adesso.anki.battle.world.bodies.Mine;
-import de.adesso.anki.battle.world.bodies.Vehicle;
-
-public class PutMineCommand extends Command {
-	
-	
-	public PutMineCommand () {
-	}
-	
-	public void execute(Vehicle vehicle) {
-		// TODO: set Positions}
-		if (!vehicle.isMineReady()) {
-			return;
-		}
-		World world = vehicle.getWorld();
-		Mine mine = new Mine ();
-		mine.setPosition(vehicle.getPosition());
-		world.addBody(mine);
-		mine.setWorld(world);
-		vehicle.setMineReady(false);
-	}
-	
-	
-}
+package de.adesso.anki.battle.protocol.commands;
+
+import de.adesso.anki.battle.world.World;
+import de.adesso.anki.battle.world.bodies.Mine;
+import de.adesso.anki.battle.world.bodies.Vehicle;
+
+public class PutMineCommand extends Command {
+	
+	@Override
+    public void execute(Vehicle vehicle) {
+		// TODO: set Positions
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=22&issues=AWKWIFf01B6WA4TguGK-&open=AWKWIFf01B6WA4TguGK-)

[ISSUE_KEY]: AWKWIFf01B6WA4TguGK-